### PR TITLE
prometheus installed by endpoint operator

### PR DESCRIPTION
**Pull request by marcolan018**
https://github.com/open-cluster-management/backlog/issues/15213
https://github.com/open-cluster-management/backlog/issues/15216

### CODE REVIEW

File: operators/endpointmetrics/manifests/prometheus/prometheus-roleBinding-default.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  annotations:
+    update-namespace: "false"
+  name: prometheus-k8s
+  namespace: default
```
Comment (morvencao) [inline operators/endpointmetrics/manifests/prometheus/prometheus-roleBinding-default.yaml:7]: Is this namespace `default` correct?

File: operators/endpointmetrics/manifests/prometheus/prometheus-role-default.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  annotations:
+    update-namespace: "false"
+  name: prometheus-k8s
+  namespace: default
```
Comment (morvencao) [inline operators/endpointmetrics/manifests/prometheus/prometheus-role-default.yaml:7]: Also this namespace `default`, is it intent?

Comment (marcolan018) [reply operators/endpointmetrics/manifests/prometheus/prometheus-role-default.yaml:7]: @morvencao 
both namespace "default" is desired.
it's from https://github.com/prometheus-operator/kube-prometheus/blob/996d5163639371925e101d24e36406871991b8b1/manifests/prometheus-roleSpecificNamespaces.yaml#L12

Comment (morvencao) [reply operators/endpointmetrics/manifests/prometheus/prometheus-role-default.yaml:7]: OK

File: operators/endpointmetrics/pkg/util/obj_compare.go [hunk 3]
```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2021 Red Hat, Inc.
```
[the remark this answers is not in the document]
Comment (clyang82) [reply operators/endpointmetrics/pkg/util/obj_compare.go:1]: You may update in another PR. let us have it in firstly

**Pull request merged**